Sources/MapboxNavigation/InstructionPresenter.swift

```diff
@@ -168,34 +171,52 @@ class InstructionPresenter {
         return attributedTextRepresentations.joined(separator: separator)
     }
     
-    func attributedString(forShieldComponent shield: VisualInstruction.Component.ImageRepresentation,
-                          repository: ImageRepository,
+    func attributedString(for representation: VisualInstruction.Component.ImageRepresentation,
```

Comment:
It follows the same logic as in `WayNameLabel` to check whether the Mapbox designed shield or the legacy shield would be used